Sorting/JS/JS_basic_flexible.js

```diff
@@ -0,0 +1,26 @@
+
+
+flexible & scalable rather than hard coding
+尽量让代码可以复用，不受个别例子影响
```

Comment:
delete empty lines. 
add comment in front of line 3 and 4: // xxxx

Reply:
done